src/cuda/tests.rs

```diff
@@ -0,0 +1,18 @@
+use crate::a_piece::CORRECT_ENCODING;
+use crate::cuda::*;
+
+#[cfg(feature = "cuda")]
```

Comment:
This feature shouldn't be on a specific test, it should be on the whole `cuda` module